.eslintrc.json

```diff
@@ -10,6 +10,14 @@
         "arrow-spacing": ["error", { "before": true, "after": true }],
         "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
         "constructor-super": "error",
+        "indent": ["error", "tab", {
```

Comment:
In what ways does this deviate from the "normal" indent rule? Or, why do we need the complicated object config?